.github/workflows/ci.yml

```diff
@@ -39,16 +39,45 @@ jobs:
             reqs/requirements-dev-test.txt
       - name: Install dependencies
         run: pip install -r reqs/requirements-dev-test.txt
+      - name: Get docs files that have changed
+        id: changed-files
+        uses: dorny/paths-filter@v2
+        with:
```

Comment:
Github action that returns changed files. We are currently only using it for `docs` changes